src/ln/functional_test_utils.rs

```diff
@@ -827,7 +827,7 @@ pub fn fail_payment(origin_node: &Node, expected_route: &[&Node], our_payment_ha
 	fail_payment_along_route(origin_node, expected_route, false, our_payment_hash);
 }
 
-pub fn create_network(node_count: usize) -> Vec<Node> {
+pub fn create_network(node_count: usize, node_config: &[Option<UserConfig>]) -> Vec<Node> {
```

Comment:
nit: If we're just gonna call UserConfig::new() here, why not just do that everywhere instead of None, None? Its not much different and keeps things obvious.

Reply:
Hmmm but if we pass a new UserConfig, then we set default flags in create_network but for some specific tests I want to deactivate them ?

Reply:
Ehh, I meant just drop the Option<>

Reply:
Wait if I pass create_network(2, UserConfig::new(), UserConfig::new()) create_network is going to set default flags, a behavior I wish to opt-out?

Reply:
You can provide custom ones instead where needed?

Reply:
I can, but if create_network override them with the default ones, how I signal to do it with a per-node granularity ?

Reply:
Keep the array? Anyway, not worth fixing if its just this left to fix.